cdap-docs/reference-manual/source/release-notes.rst

```diff
@@ -29,59 +29,52 @@ Cask Data Application Platform Release Notes
-  Added an example for Workflow fork and condition nodes.
+- `CDAP-2318 <https://issues.cask.co/browse/CDAP-2318>`__ -
+  Made CDAP work on the HDP 2.2 distribution.
 
-- `CDAP-1993 <https://issues.cask.co/browse/CDAP-1993>`__ -
-  Added ability to manipulate the SparkConf.
+- `CDAP-2786 <https://issues.cask.co/browse/CDAP-2786>`__ -
+  Added support to CDAP 3.1.0 for the MapR distro.
 
-- `CDAP-1998 <https://issues.cask.co/browse/CDAP-1998>`__ -
-  Removed deprecated Context - Query param in Metrics v3 API.
+- `CDAP-2798 <https://issues.cask.co/browse/CDAP-2798>`__ -
+  Added Hive 0.14 support.
 
-- `CDAP-2139 <https://issues.cask.co/browse/CDAP-2139>`__ -
-  Removed a duplicate Table of Contents on the Documentation Search page.
+- `CDAP-2801 <https://issues.cask.co/browse/CDAP-2801>`__ -
+  Added CDH 5.4 Hive 1.1 support.
 
-- `CDAP-2318 <https://issues.cask.co/browse/CDAP-2318>`__ -
-  Made CDAP work on the HDP 2.2 distribution.
+- `CDAP-2836 <https://issues.cask.co/browse/CDAP-2836>`__ -
+  Added support for restart of specific CDAP System Services Instances.
 
-- `CDAP-2466 <https://issues.cask.co/browse/CDAP-2466>`__ -
-  Added CLI for metrics search and query.
+- `CDAP-2853 <https://issues.cask.co/browse/CDAP-2853>`__ -
+  Completed certification process for MapR on CDAP.
 
-- `CDAP-2515 <https://issues.cask.co/browse/CDAP-2515>`__ -
-  Added a metrics client for search and query by tags.
+- `CDAP-2879 <https://issues.cask.co/browse/CDAP-2879>`__ -
+  Added Hive 1.0 in Standalone.
 
-- `CDAP-2582 <https://issues.cask.co/browse/CDAP-2582>`__ -
-  Documented the licenses of the shipped CDAP-UI components.
+- `CDAP-2881 <https://issues.cask.co/browse/CDAP-2881>`__ -
+  Added support for HDP 2.2.x.
 
-- `CDAP-2595 <https://issues.cask.co/browse/CDAP-2595>`__ -
-  Added data modelling of flows.
+- `CDAP-2891 <https://issues.cask.co/browse/CDAP-2891>`__ -
+  Documented cdap-env.sh and settings OPTS for HDP 2.2.
 
-- `CDAP-2596 <https://issues.cask.co/browse/CDAP-2596>`__ -
-  Added data modelling of MapReduce.
+- `CDAP-2898 <https://issues.cask.co/browse/CDAP-2898>`__ -
+  Added Hive 1.1 in Standalone.
 
-- `CDAP-2617 <https://issues.cask.co/browse/CDAP-2617>`__ -
-  Added the capability to get logs for a given time range from CLI.
+- `CDAP-2953 <https://issues.cask.co/browse/CDAP-2953>`__ -
+  Added HiveServer2 support in a secure cluster.
 
-- `CDAP-2618 <https://issues.cask.co/browse/CDAP-2618>`__ -
-  Simplified the Cube sink configurations.
 
-- `CDAP-2670 <https://issues.cask.co/browse/CDAP-2670>`__ -
-  Added Parquet sink with time partitioned file dataset.
+**Spark**
+
+- `CDAP-344 <https://issues.cask.co/browse/CDAP-344>`__ -
+  Users can now run Spark in Distributed mode.
+
+- `CDAP-1993 <https://issues.cask.co/browse/CDAP-1993>`__ -
+  Added ability to manipulate the SparkConf.
 
 - `CDAP-2700 <https://issues.cask.co/browse/CDAP-2700>`__ -
   Added the ability to Spark programs of discovering CDAP services.
```

Comment:
Added the ability to Spark programs of discovering CDAP services in distributed mode.